metadata/src/main/java/org/apache/kafka/metadata/storage/Formatter.java

```diff
@@ -125,7 +125,7 @@ public class Formatter {
     /**
      * The metadata log directory.
      */
-    private String metadataLogDirectory = null;
+    private Optional<String> metadataLogDirectory = null;
```

Comment:
Pardon me, why we can't have empty `metadataLogDirectory` (`Optional.empty`) by default?

Reply:
ok